documentation/api/reference/healthcheck.md

```diff
@@ -0,0 +1,21 @@
+# Django Healthcheck
+
+The Django API healthcheck endpoint checks the following:
+
+- Whether the API is accessible at all (i.e., is Django returning responses)
+- Whether the Postgres database is accessible to Django, by checking if
+  connections are available to the database
+- Optionally, whether Elasticsearch is accessible to Django (only when the
```

Comment:
```suggestion
- Optionally, whether Elasticsearch is accessible to Django (only when the
```

Is this enabled in production/by default and should we mention that here?

Reply:
We do not use it in the current production healthchecks, as far as I know.